portfolio/src/main/webapp/index.html

```diff
@@ -168,11 +111,34 @@ <h3>Shpe</h3>
         </button>
     </div>
 
-    <footer> 
-        <h5>
-            Copyright of Sofi
-        </h5>
-    </footer>
+    <div class="comment-container">
+      <h1 id="Comment">Comments</h1>
```

Comment:
The id should not be capitalized. Also, can we use "comments" instead? It should be plural.